Dockerfile

```diff
@@ -1,5 +1,11 @@
-FROM bitnami/minideb:buster
+FROM bitnami/minideb:buster as build
+RUN install_packages ca-certificates
+RUN mkdir /workdir
+
+FROM scratch
 ARG IMAGE_VERSION
 ENV IMAGE_VERSION=${IMAGE_VERSION}
+COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=build /workdir /tmp
```

Comment:
should we add a comment saying that this a workaround to create a `/tmp` folder in a scratch container?

Reply:
added